internal/store/pod.go

```diff
@@ -90,9 +90,99 @@ func podMetricFamilies(allowAnnotationsList, allowLabelsList []string) []generat
 		createPodStatusScheduledFamilyGenerator(),
 		createPodStatusScheduledTimeFamilyGenerator(),
 		createPodStatusUnschedulableFamilyGenerator(),
+		createPodReadinessProbeFamilyGenerator(),
+		createPodLivenessProbeFamilyGenerator(),
+		createPodPrivilegedFamilyGenerator(),
 	}
 }
 
+func createPodPrivilegedFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_container_privileged",
+		"Describes whether the containers are in privileged mode.",
+		metric.Gauge,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			ms := make([]*metric.Metric, len(p.Spec.Containers))
+			labelKeys := []string{"container"}
+
+			for i, c := range p.Spec.Containers {
+				isPrivileged := 0
+				if c.SecurityContext != nil && c.SecurityContext.Privileged != nil && *c.SecurityContext.Privileged {
+					isPrivileged = 1
+				}
+				ms[i] = &metric.Metric{
+					LabelKeys:   labelKeys,
+					LabelValues: []string{c.Name},
+					Value:       float64(isPrivileged),
+				}
+			}
+
+			return &metric.Family{
+				Metrics: ms,
+			}
+		}),
+	)
+}
+
+func createPodLivenessProbeFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_container_liveness_probe",
+		"Describes whether the containers liveness check succeeded.",
```

Comment:
According to your implementation, this isn't what is exposed. `Pod.Spec.Containers.LivenessProbe` will tell you whether a liveness probe is set up or not for this container.